_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,375 @@
+Apache Flink is not only growing when it comes to contributions and users, it is 
+also growing out of the original use cases and personas. Like the whole industry, 
+it is moving more towards business/analytics use cases that are implemented as 
+low-/no-code. The feature that represents the most within the Flink space is 
+Flink SQL. That’s why its popularity continues to grow. 
+
+Apache Flink is considered an essential building block in data architectures.  It 
+is included with other technologies to drive all sorts of use cases. New ideas pop 
+up, existing technologies establish themselves as standards for solving some aspects 
+of a problem. In order to be successful, it is important that the experience of 
+integrating with Apache Flink is as seamless and easy as possible. 
+
+In the 1.15 release the Apache Flink community made significant progress across all 
+these areas. Still those are not the only things that made it into 1.15. The 
+contributors improved the experience of operating Apache Flink by making it much 
+easier and more transparent to handle checkpoints and savepoints and their ownership, 
+making auto scaling more seamless and complete, by removing side effects of use cases 
+in which different data sources produce varying amounts of data, and - finally - the 
+ability to upgrade SQL jobs without losing the state. By continuing on supporting 
+checkpoints after tasks finished and adding window table valued functions in batch 
+mode, the experience of unified stream and batch processing was once more improved 
+making hybrid use cases way easier. In the SQL space, not only the first step in 
+version upgrades have been added but also JSON functions to make it easier to import 
+and export structured data in SQL. Both will allow users to better rely on Flink SQL 
+for production use cases in the long term. To establish Apache Flink as part of the 
+data processing ecosystem we improved the cloud interoperability and added more sink 
+connectors and formats. And yes we enabled a Scala-free runtime 
+([the hype is real](https://flink.apache.org/2022/02/22/scala-free.html)).
+
+
+## Operating Apache Flink with joy
+
+Even jobs that have been built and tuned by the best engineering teams still need to 
+be operated. Looking at the lifecycle of Flink based projects most of them are built 
+to stay, putting long-term burdens on the people operating them. The many deployment 
+patterns, APIs, tuneable configs, and use cases covered by Apache Flink come at the 
+high cost of support.
+
+
+### Clarifying checkpoint and savepoint semantics
+
+An essential cornerstone of Flink’s fault tolerance strategy is based on 
+[checkpoints](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/checkpoints/)
+[and](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/checkpoints_vs_savepoints/) 
+[savepoints](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/savepoints/). 
```

Comment:
```suggestion
[checkpoints](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/checkpoints/) and 
[savepoints](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/savepoints/) (see [the comparison](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/checkpoints_vs_savepoints/). 
```
nit: I would be concerned that putting three links behind each other doesn't make it clear from the start that those are actually three links.

Reply:
@XComp we don't have the same understanding of linking humour. :)